### ISSUE TITLE
Backlight turning on with "screen off" screensaver

### ISSUE THREAD

**Issue by slartibartfast3** [ralph-irving/squeezeos#3]
This has been a longstanding issue with the Touch and the Controller where occasionally the backlight will come on when the screen off screensaver is being used. When this happens waking the screen and allowing the screensaver to kick in again fixes it until next time. It is hard to say what triggers this issue I used to think it was after a server restart but that doesn't seem to be the case.

**Comment by ralph-irving** [comment]
I've also experienced this issue with my Touch, but not the radio and I don't own a controller.

Unless we discover the steps to reproduce the issue it's unlikely to get fixed.

**Comment by mw9** [comment]
Let me add a question, as I was trying to replicate this on my Controller just yesterday.

There are three places that I am seeing where the ‘Screen off’ screen saver may be used:
“When playing”, “When stopped”, and “When off”.

@slartibartfast3, which of these three is affected ? Or is there somewhere else ?

I have yet to see my ‘Screen off’ screen saver come into action with “When off”. Perhaps I should be more patient.

**Comment by slartibartfast3** [comment]
> 
> 
> Let me add a question, as I was trying to replicate this on my Controller just yesterday.
> 
> There are three places that I am seeing where the ‘Screen off’ screen saver may be used:
> “When playing”, “When stopped”, and “When off”.
> 
> @slartibartfast3, which of these three is affected ? Or is there somewhere else ?
> 
> I have yet to see my ‘Screen off’ screen saver come into action with “When off”. Perhaps I should be more patient.

On my Touch I use the 'Screen Off' screensaver "When Stopped" and "When Off". It is only recently that I started to use the Off state so it seems to occur for both Stopped and Off. My Controller doesn't have a "When Off" setting so in that case it is "When Stopped".

**Comment by mw9** [comment]
> My Controller doesn't have a "When Off" setting so in that case it is "When Stopped".

Right. So the Controller only offers the "When off" setting when its beta "Audio Playback" feature is enabled. So I've learned something today.

**Comment by slartibartfast3** [comment]
> I've also experienced this issue with my Touch, but not the radio and I don't own a controller.
> 
> Unless we discover the steps to reproduce the issue it's unlikely to get fixed.

It quite often happens for me when I turn off the Touch at the end of the day. I use Alexa to shut down theTouch (via LMS Lite) and the room lights at the same time and often see the screen glowing in the darkness. I think this even happens if the Touch is already off when the off command is sent.
